### PR TITLE
docs: add-pr-template

### PR DESCRIPTION
**Pull request by WinterYukky**
### Reason for this change
Currently, there is no pull request template, and the granularity of description is left to the discretion of contributors.
Also, the following statement is required early on in which you agree to the license for the code provided.
“*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*”

### Description of changes

Created the pull request templates based on AWS CDK repository.

### Description of how you validated changes

Nothing.

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by hoegertn — APPROVED** [review]

File: .projenrc.ts [hunk 1]
```diff
@@ -27,6 +27,32 @@ const project = new awscdk.AwsCdkConstructLibrary({
...
+
+<!--What code changes did you make? Have you made any important design decisions?-->
+
+### Description of how you validated changes
+
+<!--Have you added any unit tests and/or integration tests?-->
+
+### Checklist
+
+- [ ] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/open-constructs/aws-cdk-library/blob/main/CONTRIBUTING.md)
```
Comment (hoegertn) [inline .projenrc.ts:49]: Maybe make this a bit more explicit: pr title, tests added, etc

Comment (mrgrain) [inline .projenrc.ts:49]: FWIW I don't think checklists provide much value. They become a box ticking exercise.

It's totally a good idea to call out the contribution guide though.

File: .projenrc.ts [hunk 2]
```diff
@@ -27,6 +27,32 @@ const project = new awscdk.AwsCdkConstructLibrary({
...
+
+---
+
+_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license_
```
Comment (mrgrain) [inline .projenrc.ts:53]: Projen has a feature to enforce the contributor statement on every PR:

https://projen.io/docs/api/github?_highlight=contribu#projen.github.PullRequestLintOptions.property.contributorStatement

Comment (WinterYukky) [reply .projenrc.ts]: I didn't know it. Thank you!

**Comment by WinterYukky** [comment]
I also think a minimal checklist is fine. I think it should be automatically verified whether tests etc. have been added.

**Review by hoegertn — APPROVED** [review]

**Review by dbartholomae — APPROVED** [review]

**Pull request merged**